src/redux/reducers/quotesReducer.js

```diff
@@ -0,0 +1,40 @@
+import { FETCH_QUOTES, NEW_QUOTE } from '../actions/constants';
+
+const initialState = {
+  loading: true,
+  error: false,
+  data: [],
+  randomNumber: '',
+  colors: [
```

Comment:
It's better If the **colors** variable is defined as a constant in the application configuration file.